<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuration Files</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Singletons</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTabWidget" name="tabWidget_2">
         <property name="tabPosition">
          <enum>QTabWidget::TabPosition::West</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>ImportSettings</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3"/>
         </widget>
         <widget class="QWidget" name="tab_4">
          <attribute name="title">
           <string>ProcessingSettings</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4"/>
         </widget>
         <widget class="QWidget" name="tab_5">
          <attribute name="title">
           <string>ExportSettings</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_5"/>
         </widget>
         <widget class="QWidget" name="tab_6">
          <attribute name="title">
           <string>EngineResourceData</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_6"/>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Lists</string>
      </attribute>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>0</y>
         <width>321</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 26pt &quot;Microsoft YaHei UI&quot;;</string>
       </property>
       <property name="text">
        <string>Not Implemented</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>401</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>I'm sorry,but this function is not implemented.</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
